### DISCUSSION TITLE
Negate conditional styling?

### DISCUSSION THREAD

**Discussion by treehousekingcomic** [picocss/pico#580] [Help]
I found this [issue](https://github.com/picocss/examples/issues/19) and this [post](https://github.com/picocss/pico/discussions/441).

Is there a way to achieve `.!pico`?

I'm doing something similar but regular conditional styling won't help because my whole site uses PicoCSS including for the main container and layout so any children will always get PicoCSS. However, Pico breaks a library I'm using which creates and inserts its own elements thus I'm trying to find a way to exclude PicoCSS from a specific container which itself might be in a parent with Pico.

**Comment by dev-willis** [comment]
I have also had trouble keeping Pico out of libraries that have their own styles. The best I've come up with is this:

```
@use "@picocss/pico/scss/pico" with (
	$parent-selector: "body *:not(#library-container *) ",
);
```

This helps but does not fully prevent Pico from styling within the library's element. I'm not sure if I'm doing it wrong or if not everything in Pico respects the `$parent-selector` property.
